<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNewFactor</class>
 <widget class="QDialog" name="DialogNewFactor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>301</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>271</width>
     <height>235</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,5">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>因子编码：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditCode"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="5,5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>因子名称：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditName"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,5">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>寄存器地址：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditAddr"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="5,5">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>数据类型：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxDataType">
        <item>
         <property name="text">
          <string>int16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>uint16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>int32</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>uint32</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>float</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="5,5">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>字节序：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxByteOrder">
        <item>
         <property name="text">
          <string>ABCD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CDAB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BADC</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DCBA</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="5,5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>功能码：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxFunc">
        <item>
         <property name="text">
          <string>01</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>02</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>03</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>04</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>标识:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxTagID">
        <item>
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>属性：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxProp">
        <item>
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogNewFactor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogNewFactor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
